web/components/templates/requests/chat.tsx

```diff
@@ -352,9 +352,25 @@ export const Chat = (props: ChatProps) => {
 
   const [open, setOpen] = useState(false);
 
-  const requestMessages =
+  let requestMessages =
     llmSchema?.request.messages ?? requestBody?.messages ?? [];
 
+  const systemMessage = {
+    id: crypto.randomUUID(),
+    role: "system",
+    content: requestBody.system,
```

Comment:
Prob need a requestBody?.system here. Since we aren't checking for it before

Reply:
good catch, fixed